src/services/AccountService.ts

```diff
@@ -32,7 +32,7 @@ export class AccountService {
      * Default account derivation path
      * @var {string}
      */
-    private static readonly DEFAULT_ACCOUNT_PATH_MAIN_NET = `m/44'/4343'/0'/0'/0'`;
+    private static readonly DEFAULT_ACCOUNT_PATH_MAIN_NET = `m/44'/10111'/0'/0'/0'`;
```

Comment:
that is not a good idea, considering that it will break everyone's wallet access

Reply:
It only takes the BIP32 path when creating the profile, saved profiles will be accessible normally (as private keys are stored locally).